docs/admin/serving/config-ha.md

```diff
@@ -2,6 +2,8 @@
 title: "Configuring high-availability components"
 weight: 50
 type: "docs"
+aliases:
+  - /docs/serving/config-ha
```

Comment:
I think this documentation actually applies to _both_ serving and eventing; it was done for serving first, but this is a case where our Conway's Law docs structure actually hurt us.

Reply:
I'll move it to the top level; admin guide > HA